x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/policy_list/policy_details/policy_details.tsx

```diff
@@ -180,7 +242,7 @@ export const PolicyDetails: React.FunctionComponent<Props> = ({ policyName, onCl
       data-test-subj="policyDetail"
       aria-labelledby="srPolicyDetailsFlyoutTitle"
       size="m"
-      maxWidth={400}
+      maxWidth={550}
```

Comment:
I bumped up the max width of all flyouts in this app as [recent changes in EUI](https://github.com/elastic/eui/pull/2125) made them too narrow